R/abundance_colocalization_plot.R

```diff
@@ -0,0 +1,183 @@
+
+#' Create an abundance/colocalization scatterplot
+#'
+#' Create a scatter plot, of the abundance markers from a Seurat object. The points (each corresponding to a
+#' cell) in the scatter plot are colored by the colocalization of the two markers.
+#'
+#' @param object A Seurat object.
+#' @param markers_x A character vector of markers to plot along the x-axis.
+#' @param markers_y A character vector of markers to plot along the y-axis.
```

Comment:
Maybe we should call these `marker_1` / `marker_2` as they are called in the colocalization table?

Here it can be useful to combine the @param doc in one sentence since the two take the same type of input.

```suggestion
#' @param markers_x,marker_y A character vector of markers to plot along the x- or y-axis.
```